slacko.opam

```diff
@@ -7,14 +7,15 @@ doc: "https://leonidas-from-xiv.github.io/slacko/"
 build: [["dune" "build" "-p" name "-j" jobs]]
 run-test: [make "test"]
 depends: [
-  "dune" {build}
+  "dune" {build & >= "1.2.0"}
```

Comment:
Heh, thanks for catching it. It should've depended on 1.1 before or the `dune-project` should've stated 1.0, so that's good to have fixed now.